src/pybind/mgr/dashboard/frontend/cypress/integration/pools/pools.po.ts

```diff
@@ -40,10 +40,19 @@ export class PoolPageHelper extends PageHelper {
 
     cy.get('input[name=pgNum]').clear().type(`${new_pg}`);
     cy.get('cd-submit-button').click();
-    const str = `${new_pg} active+clean`;
     this.getTableRow(name);
     if (wait) {
-      this.getTableRow(name).contains(str);
+      this.getTableRow(name)
+        .get('.datatable-body-cell')
+        .eq(4)
+        .text()
+        .should((pgStatus) => {
+          expect(
+            pgStatus
+              .split(',')
+              .reduce((acc, curr) => (acc += curr ? parseInt(curr.trim().split(' ')[0], 10) : 0), 0)
+          ).to.equal(new_pg);
```

Comment:
How is this filtering out non-active/clean PGs?

Reply:
@avanthakkar ^^ in case you missed this comment ;-)

Reply:
Yeah, I tried with no filters to cover all cases (undersized, unknown, etc), just to see if it passes in that case. But when cypress runs another try it's already `8 active+clean` and that's why e2e failed. In any case I will include only the active ones, but still it won't fix anything, tests will remain flaky.